brapi.yaml

```diff
@@ -1181,6 +1181,391 @@ paths:
+        style: simple
+        explode: false
+        schema:
+          pattern: ^Bearer .*$
+          type: string
+        example: Bearer XXXX
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MethodsResponse'
+              example:
+                metadata:
+                  datafiles: []
+                  pagination:
+                    currentPage: 0
+                    pageSize: 2
+                    totalCount: 4
+                    totalPages: 2
+                  status: []
+                result:
+                  data:
+                  - class: Numeric
+                    description: Standard rolled measuring tape
+                    formula: a^2 + b^2 = c^2
+                    methodDbId: m1
+                    methodName: Tape Measure
+                    name: Tape Measure
+                    ontologyReference:
+                      documentationLinks:
+                      - URL: https://ontology.org/m1
+                        type: RDF
+                        url: https://ontology.org/m1
+                      ontologyDbId: MO_123
+                      ontologyName: Ontology.org
+                      version: "17"
+                    reference: google.com
+                  - class: Numeric
+                    description: Dried sample on electric scale
+                    formula: NA
+                    methodDbId: m2
+                    methodName: Dry Electric Scale
+                    name: Dry Electric Scale
+                    ontologyReference:
+                      documentationLinks:
+                      - URL: https://ontology.org/m2
+                        type: WEBPAGE
+                        url: https://ontology.org/m2
+                      ontologyDbId: MO_123
+                      ontologyName: Ontology.org
+                      version: "17"
+                    reference: google.com
+        400:
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                type: string
+              example: |-
+                ERROR - 2018-10-08T20:15:11Z - Malformed JSON Request Object
+                ERROR - 2018-10-08T20:15:11Z - Invalid query parameter
+                ERROR - 2018-10-08T20:15:11Z - Required parameter is missing
+        401:
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                type: string
+              example: ERROR - 2018-10-08T20:15:11Z - Missing or expired authorization
+                token
+        403:
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                type: string
+              example: ERROR - 2018-10-08T20:15:11Z - User does not have permission
+                to perform this action
+    post:
```

Comment:
I didn't see any code for updating the database, so this doesn't appear to be supported.